### ISSUE TITLE
Remote media cache cleanup not working

### ISSUE THREAD

**Issue by ThomasLeister** [tootsuite/mastodon#9192]
### Expected behaviour

Mastodon should delete outdated media cache files

### Actual behaviour

Error message:

```
mastodon@mastodon:~/live$ export RAILS_ENV=production
mastodon@mastodon:~/live$ cd /home/mastodon/live && /home/mastodon/.rbenv/shims/bundle exec rake mastodon:media:remove_remote
rake aborted!
Don't know how to build task 'mastodon:media:remove_remote' (see --tasks)
/home/mastodon/live/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/home/mastodon/.rbenv/versions/2.5.3/bin/bundle:23:in `load'
/home/mastodon/.rbenv/versions/2.5.3/bin/bundle:23:in `<main>'
(See full trace by running task with --trace)
```

### Steps to reproduce the problem

Execute the above command. I've tried this on two different Mastodon instances - both are affected. Is remote media cleanup still a thing? It's not mentioned in the new Mastodon Admin Docs: https://docs.joinmastodon.org/administration/installation/

### Specifications

Mastodon 2.6.1 on Ruby 2.5.3

**Comment by ThibG** [comment]
Indeed, the rake task is gone, it has been replaced by `tootctl media remote`

**Comment by gnppn** [comment]
Hi!

Thanks for the issue and the answer, which helped me. Is it possible to update [the Production guide](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Production-guide.md) to reflect this?

Thanks again!

**Comment by nightpool** [comment]
Note: The documentation has moved to a different repository
<https://source.joinmastodon.org/mastodon/docs>, where it is compiled to
docs.joinmastodon.org. This repository is no longer updated.

**Comment by gnppn** [comment]
@nightpool Yes, I only saw the documentation repo has been archived after my (now removed) comment. I feel really silly now, sorry for the noise.

**Comment by nightpool** [comment]
no worries

**Comment by nightpool** [comment]
(i normally respond/read github issues via email, so I didn't see the deletion)

**Comment by quimgil** [comment]
> tootctl media remove was added in 2.5.0, now in 2.6.0 the old rake task was removed

Does this mean that https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Production-guide.md#remote-media-attachment-cache-cleanup needs to be updated?
